common/source/docs/common-omnibusnanov6.rst

```diff
@@ -86,17 +102,34 @@ GPS is attached to UART6
 
 Telem is available at UART 1
 
-The shared UART3/I2C pins are enabled only for I2C operation to allow external compass or digital airspeed sensor attachment.
+The shared UART3/I2C pins are enabled for I2C operation in default :ref:`BRD_ALT_CONFIG <BRD_ALT_CONFIG>` = 0 to allow external compass or digital airspeed sensor attachment.
+Alternatively they can be set to UART operation as SERIAL4 in :ref:`BRD_ALT_CONFIG <BRD_ALT_CONFIG>` = 1.
 
 RC input is via pad marked LED on the board and is compatible all RX serial protocols supported by ArduPilot.
 The Buzzer output pad has no functionality.
 
+V6.x revision
+-------------
+
+.. note::
+
+   This board revision has the fixed inverter moved to USART1 RX pin, so only TX1 pin can be used (output only). 2 full UARTS + I2C is available in BRD_ALT_CONFIG = 1, using RX3 / TX3 pins as SERIAL4. In default config, RX3 and TX3 will default to I2C. RCInput is on WSL2812 pad. I2C and SERIAL3 for GPS / mag combo is on PPM / PB9 / RX6 / TX6 pads. Make sure to eave solder jumpers for inverter setup diconnected.
+
+
+.. image:: ../../../images/OmnibusF4NanoV62_pinout.jpg
+    :target: ../_images/OmnibusF4NanoV62_pinout.jpg
+    :width: 450px
+
 Battery monitoring
 ==================
 
-Supports analog voltage monitoring on VBAT pin
+Original V6 revision 
+--------------------
+
+Original V6 revision supports analog voltage monitoring on VBAT pin.
+
+Set :ref:`BATT_MONITOR <BATT_MONITOR>` to 3 (= analog voltage only) and reboot.
 
-Set :ref:`BATT_MONITOR <BATT_MONITOR>` to 3 (= analog voltage only) and reboot
```

Comment:
the comment about rebooting after setting BATT_MONITOR needs to be in both setup flows...its a common error that people complain about not seeing the other params....while the defaults should already be there on first firmware load...someone changing vehicles, etc. will come back to this section and run into it ...the comment could be:
note, if these param are not visible afte setting or changing BAT_MONITOR, reboot the autopilot...

Reply:
check lines 131 / 152, i think it's there in both